app/dashboard/templates/shared/profile_activities_bounty.html

```diff
@@ -1,4 +1,7 @@
-<div class="row result bounty_row {{ bounty.network }}" {% if bounty.github_url %}href="{{ bounty.github_url }}"{% endif %}>
+<a class="row result bounty_row {{ bounty.network }}" target="_blank"
+ {% if bounty.url %} href="{{ bounty.url }}"
+ {% elif bounty.github_url %} href="{{ bounty.github_url }}"
+ {% endif %} </a>
```

Comment:
Looks like this `<a` never closes?

Reply:
Ah that was cause of the editor auto complete 😅  -> It's supposed to be `>` instead of `</a>`